view_all_inc.php

```diff
@@ -111,10 +111,15 @@
 		<div class="btn-toolbar">
 			<div class="btn-group pull-left">
 		<?php
-			# -- Print and Export links --
+			# -- Print link --
```

Comment:
@vboctor : can you please give us your thoughts about a second threshold as suggested by @atrol ?